### PR TITLE
show tooltip when name/desc is truncated

### PR DESCRIPTION
**Pull request by myTechPlayArea**
image

### CODE REVIEW

**Comment by myTechPlayArea** [comment]
@natabal can you please review when you get a chance

**Comment by natabal** [comment]
@myTechPlayArea Can you merge this and bring in master today?

File: Graphs/TreeGraph/index.js [hunk 1]
```diff
@@ -474,6 +475,23 @@ class TreeGraph extends AbstractGraph {
 
         // ****************** Nodes section ***************************
 
+        const showToolTip = (d, showToolTip) => {
```
Comment (natabal) [inline Graphs/TreeGraph/index.js]: these should not be inner functions! Move outside

File: Graphs/TreeGraph/index.js [hunk 2]
```diff
@@ -474,6 +475,23 @@ class TreeGraph extends AbstractGraph {
...
+                tooltip.transition()
+                    .duration(200)
+                    .style("opacity", 5)
+                tooltip.html(`<small style="font-weight:bold; font-size:10px; word-wrap: break-word; text-align: center; padding: 4px; color: #000000;">${d.data.name}</small><br/><small style="text-align: justify;word-wrap: break-word;margin-top: 10px;font-size: 9px;padding: 4px;color: #000000;">${(d.data.description) ? d.data.description : commonEN.general.noDescription}</small>`)
+                    .style("left", (d3.event.pageX) + "px")
+                    .style("top", (d3.event.pageY - 28) + "px");
+            }
+        }
+
+        const hideToolTip = (d) => {
```
Comment (natabal) [inline Graphs/TreeGraph/index.js]: same with this one!

Comment (myTechPlayArea) [reply Graphs/TreeGraph/index.js]: done

File: Graphs/TreeGraph/index.js (hunk 1 above)
Comment (myTechPlayArea) [reply Graphs/TreeGraph/index.js]: done

**Pull request merged**